src/todoInput.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+
+function TodoInput({ onSubmit }) {
+  const [value, setValue] = useState('');
+
+  function handleChange(event) {
+    setValue(event.target.value);
+  }
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    onSubmit(value);
+    setValue('');
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input type="text" value={value} onChange={handleChange} />
```

Comment:
```suggestion
      <input 
        type="text"
        value={value}
        onChange={handleChange}
       />
```

속성이 많은 경우 코드 가독성을 고려해서 줄바꿈해주시면 좋아요.